src/app/shared/main-layout/main-layout.component.ts

```diff
@@ -1,8 +1,27 @@
-import { Component } from "@angular/core";
+import { Component, OnInit } from "@angular/core";
+import { Observable } from "rxjs";
+import { debounceTime } from "rxjs/operators";
+import { LoadingService } from "../../core/services/loading-service/loading-service.service";
 
 @Component({
     selector: "app-main-layout",
     templateUrl: "./main-layout.component.html",
     styleUrls: ["./main-layout.component.scss"]
 })
-export class MainLayoutComponent {}
+export class MainLayoutComponent implements OnInit {
+    private loadingService: LoadingService;
+
+    /**
+     * Observable that is used to store Loader status
+     */
+    public loading: Observable<boolean>;
```

Comment:
is loading and loading$ if it is an observable

Reply:
fixed